types/base64util/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+  "compilerOptions": {
+      "module": "commonjs",
+      "lib": [
+          "es6"
+      ],
+      "noImplicitAny": true,
+      "noImplicitThis": true,
+      "strictNullChecks": true,
+      "strictFunctionTypes": true,
+      "baseUrl": "../",
+      "typeRoots": [
+          "../"
+      ],
+      "types": [],
+      "noEmit": true,
+      "forceConsistentCasingInFileNames": true,
+      "esModuleInterop": true
```

Comment:
Can you remove `esModuleInterop` from this config? We try to adhere to a standard set of compiler options in DT whenever possible, and this option is not in the standard.

Reply:
For sure. I'll update it asap.